packages/components/src/public/QueryModel/withQueryModels.tsx

```diff
@@ -476,10 +491,13 @@ export function withQueryModels<Props>(
             );
         };
 
-        loadRows = async (id: string): Promise<void> => {
+        loadRows = async (id: string, loadSelections = false): Promise<void> => {
             const { loadRows } = this.props.modelLoader;
+            const currentModel = this.state.queryModels[id];
```

Comment:
nit: In general throughout this file we fully dereference `this.state.queryModels` so we don't accidentally process the wrong state.

Reply:
Ah yes, thanks. I'll change that.